app/next-client-app/components/scanreports/ButtonsRow.tsx

```diff
@@ -5,9 +5,11 @@ import { BookText, ChevronRight, Pencil } from "lucide-react";
 export function ButtonsRow({
   scanreportId,
   tableId,
+  permissions,
 }: {
   scanreportId: string;
   tableId: string;
+  permissions: string[];
```

Comment:
Should this be of type `Permission[]`?

https://github.com/Health-Informatics-UoN/CaRROT-Mapper/blob/8d9b83936c438415caabc1073306261c50e57fdb/app/next-client-app/types/permissions.ts